### DISCUSSION TITLE
Add support for DeArrow?

### DISCUSSION THREAD

**Discussion by kitsah** [YTLitePlus/YTLitePlus#297] [Ideas]
DeArrow is a browser extension made by the same developers as SponsorBlock.

It replaces clickbaity Thumbnails and titles with better ones. I think it would be a game changer to implement this into YTLitePlus

There is a DeArrow API available for free which YTLitePlus can tap into.

Demo of DeArrow: https://youtu.be/QjjpDhHh_QI?si=YRYkPFHTMZrCBQtu
API docs: https://wiki.sponsor.ajay.app/w/API_Docs/DeArrow

**Comment by billchase2** [comment]
I found this tweak while specifically looking for one with DeArrow support. It would make an excellent addition!
